front-end/.nvmrc

```diff
@@ -0,0 +1 @@
+v16.15.1
```

Comment:
How did you generate this? You ran like, `docker compose up run --rm front end bash -c "rm -rf node_modules && yarn install"`? Otherwise we're using a version of nvm on your local and potentially not the image.

Reply:
Or do we even have nvm on the image.. this is just for local or something? The point is, never ever generate your node modules from your local terminal, always be in a container.

Reply:
```
❯ docker compose run --rm front-end node --version
[+] Running 6/0
 ⠿ Container arad-database-1          Created                                                                                          0.0s
 ⠿ Container arad-be-administrator-1  Created                                                                                          0.0s
 ⠿ Container arad-be-reviewer-1       Created                                                                                          0.0s
 ⠿ Container arad-be-identity-1       Created                                                                                          0.0s
 ⠿ Container arad-be-reader-1         Created                                                                                          0.0s
 ⠿ Container arad-api-1               Created                                                                                          0.0s
[+] Running 6/6
 ⠿ Container arad-database-1          Healthy                                                                                          5.8s
 ⠿ Container arad-be-administrator-1  Healthy                                                                                          5.7s
 ⠿ Container arad-be-identity-1       Healthy                                                                                          6.2s
 ⠿ Container arad-be-reviewer-1       Healthy                                                                                          6.2s
 ⠿ Container arad-be-reader-1         Healthy                                                                                          5.7s
 ⠿ Container arad-api-1               Started                                                                                          0.3s
v18.5.0
```